docs/tlspolicy/tls-policy.md

```diff
@@ -151,10 +152,10 @@ spec:
           route53:
             hostedZoneID: <YOUR HOSTED ZONE ID>
             region: us-east-1
-            accessKeyID: <AWS_SECRET_ACCESS_KEY>
+            accessKeyID: <AWS_SECRET_ACCESS_KEY_ID>
```

Comment:
This looked a little strange to me (i.e. one part of AWS creds inline, one a secret)

Not for now, but I saw the `Issuer` docs for cert-manager mention you can also create an `accessKeyIDSecretRef`:

```
accessKeyIDSecretRef:
  name: prod-route53-credentials-secret
  key: access-key-id
```

We could probably swap to use this later on (or maybe support both)

cc: @mikenairn

Reply:
I am fine with this change landing as-is btw

Reply:
Yes, you should be able to do it that way, but for some reason it didn't work with the version of the CRDs we were installing. Meant to look into why as it would be better that way, and would allow us to use the same secret as we use for dns [see](https://github.com/Kuadrant/multicluster-gateway-controller/issues/382#issuecomment-1661872837.)

Reply:
I think the changes here look fine though, can look into why that didn't work separately.

Reply:
Example of the error i was getting ```Error from server (BadRequest): error when creating "scratch/kuadrant_tlspolicy/issuer-le-staging.yaml": Issuer in version "v1" cannot be handled as a Issuer: strict decoding error: unknown field "spec.acme.solvers[0].dns01.route53.accessKeyIDSecretRef" ```

Reply:
So we're good to keep this as is for now?

EDIT: although i should note that this change doesn't resolve the full issue of getting Let'sEncrypt setup and working correctly with MGC. Still working on that.

Reply:
Opened a PR to update this https://github.com/Kuadrant/multicluster-gateway-controller/pull/584